### PR TITLE
fume hood table updates

### PR DESCRIPTION
**Pull request by JasonZ08**
Create_Modal
Edit_Modal
Sash_Height

### CODE REVIEW

**Comment by michaelli00** [comment]
Nice job. Couple of comments

- Could you style the button to match the figma color (light blue). Create a constant in Constants.js for the hex of this color so we can reuse it later
- User should be able to update the fume hood name of an existing fume hood, so the fume hood field should be a textedit where the current value is the current fume hood name
- User shouldn't be able to edit the sensor MAC address. We use this as the id of the sensor

**Comment by JasonZ08** [comment]
image
image
image

Made button the same color as the progress bar. Do you want both of them to be the color on the figma or just the button to be the color on the figma? Added comments and also changed the modal forms. Let me know if theres anything else you want changed.

**Review by michaelli00 — COMMENTED** [review]
Sorry first set of comments didn't go through for some reason. But otherwise looks good

File: src/components/FumeHoodsPage/FumeHoodsPage.js [hunk 1]
```diff
@@ -7,174 +7,152 @@ import Row from 'react-bootstrap/Row';
...
+		setShouldShowModalForm(false)
+		setSensor(null)
   }
 
   function findCFM(sensorName, arr) {
-	let res = "HEIGHT"
-	arr.forEach((ele) => {
-		if (ele.fumeHood === sensorName) {
-			res = <ProgressBar variant='info' now={ele.normalizedCfm}/>
+		let res = arr.find((ele) => ele.fumeHood === sensorName)
```
Comment (michaelli00) [inline src/components/FumeHoodsPage/FumeHoodsPage.js:45]: We can guarantee the element will be found in the list. So we can simplify this to
```
const res = arr.find(ele => ele.fumeHood === sensorName);
return <ProgressBar variant='info' now={res.normalizedCfm}/>
```

File: src/components/FumeHoodsPage/components/FumeModalForm.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                  defaultValue='Room'
+                  required
+                />
+                <Form.Label>
+                  <h5>Lab</h5>
+                </Form.Label>
+                <Form.Control
+                  type='input'
+                  placeholder='Lab'
+                  defaultValue='Lab'
```
Comment (michaelli00) [inline src/components/FumeHoodsPage/components/FumeModalForm.js:71]: Same thing as before. I don't think we need a defaultValue? Let me know if it's actually needed

File: src/components/FumeHoodsPage/components/FumeModalForm.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+                  required
+                />
+              </Form.Group>
+            </Modal.Body>
+            <Modal.Footer>
+              <Button variant='dark' type='submit'> Update Fume Hood </Button>
```
Comment (michaelli00) [inline src/components/FumeHoodsPage/components/FumeModalForm.js:77]: Did you update the color of the button to match the figma? Light blue?  Create a .scss file and add the styling there

**Review by michaelli00 — COMMENTED** [review]
When we're done with review, can you squash the commits so that it's a single commit?

**Pull request merged**